packages/modules/itwin-viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/** Clone of core BasicNavigationWidget with conditional tooling
+ */
+import {
+  CommonToolbarItem,
+  ToolbarOrientation,
+  ToolbarUsage,
+} from "@bentley/ui-abstract";
+import {
+  CoreTools,
+  NavigationWidgetComposer,
+  ToolbarComposer,
+  ToolbarHelper,
+} from "@bentley/ui-framework";
+import * as React from "react";
+
+import { ViewNavigationTools } from "../../../types";
+import { useUiVisibility } from "./BasicToolWidget";
+
+export interface BasicNavigationWidgetProps {
+  config?: ViewNavigationTools;
+}
+
+/** Basic Navigation Widget that provides standard tools to manipulate views containing element data.
+ * Supports the specification of additional horizontal and vertical toolbar items through props.
+ */
+export function BasicNavigationWidget({ config }: BasicNavigationWidgetProps) {
+  const getHorizontalToolbarItems = React.useCallback((): CommonToolbarItem[] => {
+    if (config?.hideDefaultHorizontalItems) {
+      return [];
+    }
+    const items: CommonToolbarItem[] = ToolbarHelper.createToolbarItemsFromItemDefs(
+      [
+        CoreTools.rotateViewCommand,
+        CoreTools.panViewCommand,
+        CoreTools.fitViewCommand,
+        CoreTools.windowAreaCommand,
+        CoreTools.viewUndoCommand,
+        CoreTools.viewRedoCommand,
+      ]
+    );
+    return items;
+  }, [config]);
+
+  const getVerticalToolbarItems = React.useCallback((): CommonToolbarItem[] => {
+    if (config?.hideDefaultVerticalItems) {
+      return [];
+    }
+    const items: CommonToolbarItem[] = [];
+    items.push(
+      ToolbarHelper.createToolbarItemFromItemDef(10, CoreTools.walkViewCommand),
```

Comment:
Is the LookAndMove tool available? If so we should replace the walkViewCommand with that.